examples/cadmium-dynamic-LI33.cpp

```diff
@@ -0,0 +1,174 @@
+
+//THIS IS A REF FOR AN AUTOGENERATED MODEL
+/**
+ * Copyright (c) 2019, Juan Lanuza
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#include <chrono>
+#include "../src/cadmium-devstone-atomic.hpp"
+#include "../src/cadmium-event-reader.hpp"
+
+#include <cadmium/modeling/coupled_model.hpp>
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/dynamic_model_translator.hpp>
+#include <cadmium/concept/coupled_model_assert.hpp>
+#include <cadmium/modeling/dynamic_coupled.hpp>
+#include <cadmium/modeling/dynamic_atomic.hpp>
+#include <cadmium/engine/pdevs_dynamic_runner.hpp>
+#include <cadmium/logger/tuple_to_ostream.hpp>
+#include <cadmium/logger/common_loggers.hpp>
+
+using TIME = float;
```

Comment:
It would be nice to keep this as a parameter in the future since we plan to compare the usages of different time data types against float as a goal in the long term. However, it is fine to keep it like this for now.

Reply:
I see it cannot be merged by rebase. Try to sync your repository before next PR.

Reply:
Merged it as squash merge